libtrip-notification/src/main/res/values-ca/strings.xml

```diff
@@ -1,4 +1,8 @@
 <resources>
     <!-- Notification Strings -->
     <string name="mapbox_end_navigation">Aturar navegació</string>
-    </resources>
+    <string name="mapbox_navigation_is_starting">Comença la navegació...</string>
```

Comment:
Some languages are using `...`, while others are using a proper Unicode ellipsis (`…`). Which is more appropriate on Android?

Reply:
Unicode ellipsis relevant in Android too